client/src/VideoCards.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import videosData from "./exampleresponse.json";
+import Video from "./Videos";
+import VoteButtons from "./VoteButtons";
+import AddVideo from "./AddVideo";
+
+function VideoCards() {
+  const [videoList, setVideoList] = useState(videosData);
+
+  /* remove any video by id */
+  function removeVideo(id) {
+    const updatedVideoList = videoList.filter((video) => video.id !== id);
+    setVideoList(updatedVideoList);
+  }
+
+  /* add any video to list */
+  function addVideo(videoObj) {
+    setVideoList([videoObj, ...videoList]);
+  }
+
+  return (
+    <div className="video-cards">
+      <AddVideo AddVideo={addVideo} />
```

Comment:
the prop `AddVideo` should be in camel-case, so with a lowercase `a`. Only component names being with a capital letter
```
<AddVideo addVideo={addVideo} />
```